petastorm/tests/test_parquet_reader.py

```diff
@@ -70,6 +71,18 @@ def test_simple_read(scalar_dataset, reader_factory):
         _check_simple_reader(reader, scalar_dataset.data)
 
 
+@pytest.mark.parametrize('reader_factory', _D)
+def test_simple_read(scalar_dataset, reader_factory):
+    """See if we can read data when a single parquet file is specified instead of a parquet directory"""
+    assert scalar_dataset.url.startswith('file://')
```

Comment:
Although our fix affects only make_reader(..) readers, let's check that the same issue does not occur with make_batch_reader.

Reply:
Unit test fails due to using same function name as line 68